src/systems/fem_context.C

```diff
@@ -250,8 +259,16 @@ void FEMContext::some_interior_gradient(unsigned int var, unsigned int qp, Outpu
 
 #ifdef LIBMESH_ENABLE_SECOND_DERIVATIVES
 template<typename OutputType,
+         void (FEMContext::*fe_getter)
+           (unsigned int,
+            FEGenericBase
+              <typename TensorTools::MakeReal
+                <typename TensorTools::DecrementRank
+                  <typename TensorTools::DecrementRank
+                    <OutputType>::type>::type>::type> *&)
```

Comment:
Hmm... maybe a templated helper class (basically a c++03-compatible template typedef) would be good here?

I (tried)[https://github.com/jwpeterson/libmesh/commit/42fc9791d287f697c5c10e217968c07e01b67a40] to add one, but it doesn't compile.  Not sure if it's something fundamental or just a syntax error.